### ISSUE TITLE
Link to online Gerbalyze fails

### ISSUE THREAD

**Issue by puterboy** [jaseg/gerbolyze#33]
Clicking on https://dyna.kokoroyukuma.de/gerboweb gives a 404 error.

**Comment by jaseg** [comment]
Yeah, I might just remove that link... That website is hosted on an old laptop in my actual broom closet. Uptime hasn't been great recently. Try the command-line utilities instead, they are much more capable and recently received some improvements.
